src/components/chart.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import * as Highcharts from 'highcharts';
+
+class Chart extends React.Component<any, any> {
+  baseOptions:any = {
```

Comment:
This is a general issue, but I typed a lot of things `any`, including the types passed to `Component`. If we're going to build a TS project, I should probably learn how to write proper TS.